test/optimus/pr/ValidateOptimus.wdl

```diff
@@ -0,0 +1,62 @@
+
+task ValidateOptimus {
+      File bam
+      File matrix
+      File matrix_summary
+      File picard_metrics
+      Int required_disk = ceil((size(bam, "G") + size(matrix, "G") + size(matrix_summary, "G") + size(picard_metrics, "G")) * 1.1)
+
+      String expected_bam_hash
+      String expected_matrix_hash
+      String expected_matrix_summary_hash
+      String expected_picard_metrics_hash
+      # # these are also inputs; lets leave them alone until flattening is in WDL
+      # Array[Array[File]] tag_gene_exon_log
+      # Array[Array[File]] umi_metrics
+      # Array[Array[File]] duplicate_metrics
+
+  command <<<
+
+    # calculate hashes; awk is used to extract the hash from the md5sum output that contains both
+    # a hash and the filename that was passed
+    bam_hash=$(samtools view "${bam}" | md5sum | awk '{print $1}')
```

Comment:
it might be worth noting that we are ignoring the bam headers for the md5 (rightly so in this case!).